secure-api-gateway-ob-uk-rs-server/src/main/java/com/forgerock/sapi/gateway/ob/uk/rs/server/api/backoffice/payment/utils/PaymentConsentCalculateElementsService.java

```diff
@@ -28,17 +29,25 @@
 import com.forgerock.sapi.gateway.uk.common.shared.api.meta.share.IntentType;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
 import uk.org.openbanking.datamodel.error.OBError1;
 
 import javax.servlet.http.HttpServletRequest;
 import java.util.List;
 import java.util.Objects;
 
 @Slf4j
-public class PaymentConsentGeneral {
-    public static final String API_VERSION_DESCRIPTION = "Api version";
-    public static final String INTENT_TYPE_DESCRIPTION = "Intent type";
-    private static final CustomObjectMapper customObjectMapper = CustomObjectMapper.getCustomObjectMapper();
+@Service
+public class PaymentConsentCalculateElementsService {
+    public final String API_VERSION_DESCRIPTION = "Api version";
+    public final String INTENT_TYPE_DESCRIPTION = "Intent type";
+
+    // Setting a custom-configured ObjectMapper Bean {@see RSApplicationConfiguration#Jackson2ObjectMapperBuilderCustomizer}
+    private final ObjectMapper objectMapper;
+
+    public PaymentConsentCalculateElementsService(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
```

Comment:
Nice, this is much better than the previous impl. Allow dependencies to be wired via the constructor is much cleaner than a static lookup